kolibri-electron/src/index.js

```diff
@@ -67,9 +183,14 @@ const createWindow = () => {
     return {action: 'deny'};
   };
   mainWindow.webContents.setWindowOpenHandler(windowOpenHandler);
+  mainWindow.loadFile(await getLoadingScreen());
 
-  // Load the loading screen
-  mainWindow.loadFile(path.join(__dirname, 'Kolibri', 'assets', '_load.html'));
+  const firstLaunch = await checkVersion();
+  if (firstLaunch) {
+    mainWindow.webContents.executeJavaScript('firstLaunch()', true);
```

Comment:
I suspect there's a race condition here, as we might execute `firstLaunch` before the loading screen has loaded. The loadFile method returns a Promise which resolves after the page has loaded, so we could use that.

Reply:
I've added an `await` for the `loadFile` call, so there's no race condition here. Now the version check is done after showing the loading screen.